bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -173,6 +173,27 @@ rules:
     - services/finalizers
   verbs:
     - update
+- apiGroups: ["k8s.ovn.org"]
+  resources:
+    - userdefinednetworks
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups: ["k8s.ovn.org"]
+  resources:
+    - userdefinednetworks/status
```

Comment:
need to add `userdefinednetworks` here too, as we need patch and update on the resource to add finalizers

Reply:
Done, sorry for the mess.